tools/cockpit.spec

```diff
@@ -347,8 +347,8 @@ Cockpit support for reading PCP metrics and loading PCP archives.
 %package dashboard
 Summary: Cockpit SSH remoting and dashboard
 Requires: libssh >= %{libssh_version}
-Requires: cockpit-ws = %{version}-%{release}
-Provides: cockpit-ssh = %{version}-%{release}
+Requires: cockpit-bridge > 134
```

Comment:
Likewise, would it make sense  to bump the dependency in pkg/dashboard/manifest.json.in instead and move to `%required_base` here also?